generators/client/templates/react/package.json.ejs

```diff
@@ -186,7 +186,7 @@ limitations under the License.
   },
 <%_ } _%>
   "engines": {
-    "node": ">=8.9.0"<% if (clientPackageManager === 'yarn') { %>,
+    "node": ">=12.6.1"<% if (clientPackageManager === 'yarn') { %>,
```

Comment:
```suggestion
    "node": ">=12.16.1"<% if (clientPackageManager === 'yarn') { %>,
```

Reply:
I've updated Node to 10 instead. :smile: